README.md

```diff
@@ -1,2 +1,25 @@
 # dbluff-poc
 Proof of Concept
+
+The Proof of Concept is made up of three types of containers: traffic sources, traffic targets, and proxies.  The initial PoC will be:
```

Comment:
I would like to see the file wrap at 80 columns (excepting links).  If you use vscode, you can use the rewrap extension to easily do this for you, for example.